tethysext/atcore/resources/job_scripts/update_resource_status.py

```diff
@@ -0,0 +1,56 @@
+#!/opt/tethys-python
+import sys
+import traceback
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from tethysext.atcore.models.app_users import Resource
+
+
+def run(resource_db_url: str, resource_id: str, status_keys: list):
+    """
+    Update the status of a Resource
+
+    Args:
+        resource_db_url (str): The url to the resource in the database.
```

Comment:
"The SQLAlchemy URL to the resource database."

Reply:
fixed